Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:latest
+
+COPY . .
+RUN apt-get update && apt-get install -y redis-server && gem install bundler -v '2.0.1' && bundle update --bundler

```

Comment:
Can we move this to separate layer. Copying entire app files before installing dependencies is not a good approach.
Please see the best practices: https://medium.com/the-code-review/docker-copy-dockerfile-best-practices-503704bee69f

Reply:
Ideally this should be the docker file:

```Dockerfile
FROM ruby:latest
WORKDIR /app
COPY Gemfile Gemfile.lock ./
RUN apt-get update && apt-get install -y redis-server && gem install bundler -v '2.0.1' && bundle update --bundler
COPY . .
CMD eval 'redis-server &' && bundle install && bundle exec puma
```

Reply:
@gautamkrishnar Pefrect, thanks.